evolver/calibration/standard/calibrators/temperature.py

```diff
@@ -26,7 +32,17 @@ def create_calibration_procedure(
         *args,
         **kwargs,
     ):
-        procedure_state = CalibrationStateModel.load(self.procedure_file) if resume else None
+        procedure_state = None
+        if resume and self.procedure_file:
+            # Check if procedure_file is an absolute path
+            if Path(self.procedure_file).is_absolute():
+                procedure_file_path = self.procedure_file
+            else:
+                # Use self.dir if defined, otherwise use the default storage path
+                procedure_dir = getattr(self, "dir", root_calibrator_file_storage_path())
+                procedure_file_path = procedure_dir / self.procedure_file
```

Comment:
feels a bit like this logic belongs in the `load` method of `CalibrationStateModel`, or rather even its relevent base (I think the `Transformer.Config`.